akka-http-core/src/main/java/akka/http/javadsl/model/HttpEntity.java

```diff
@@ -142,6 +140,8 @@
      */
     CompletionStage<HttpEntity.Strict> toStrict(long timeoutMillis, Materializer materializer);
 
+    HttpMessage.DiscardedEntity discardEntityBytes(Materializer materializer);
```

Comment:
please add documentation on the method (same as on the HttpMessage version of this method)